### ISSUE TITLE
Updating Julia binaries

### ISSUE THREAD

**Issue by mipals** [flatironinstitute/FMM3D#26]
Hello,

Is there any plan to update the Julia binaries to run on the M1 chip?

Cheers,
Mikkel

**Comment by mipals** [comment]
I just found the build script:

[https://github.com/JuliaPackaging/Yggdrasil/blob/master/F/FMM3D/build_tarballs.jl](url)

Seems like it explicitly removes the M1 chips from support 😢  I guess there is some compile problems with FMM3D itself on M1 then?

**Comment by mrachh** [comment]
Dear Mikkel,
The issue was the absence of a stable Fortran compiler when the M1 chips first came out. Since it is available now, we will try and get the binaries built on M1, update the scripts and keep you posted. 

Regards,
Manas

**Comment by mipals** [comment]
Dear Manas,

That makes sense 💻  - Yes, please keep my posted 👍 

Cheers,
Mikkel

**Comment by mipals** [comment]
Hi again,

Is there any update on this?

Cheers,
Mikkel

**Comment by mipals** [comment]
Looks like things are up and running thanks a lot @askhamwhat  😄  (although I could only get it to work on Julia 1.8-RC1)

By the way, the installation can be done more easily than described in the README.md

```julia
using Pkg
Pkg.add(url="https://github.com/flatironinstitute/FMM3D.git", subdir="julia")
```

**Comment by askhamwhat** [comment]
@mipals Glad to hear it and thanks for the notes. I'm not sure why it would need a later version of Julia but I'll add a note to the README. And great, that is a cleaner method for installation. Cheers

**Comment by mipals** [comment]
The error that I got was that `libquadmath.0.dylib` could not be found. [This error was fixed and backported 1.7.3](https://github.com/JuliaLang/julia/issues/44506), but I was trying on 1.7.2 earlier. Everything works nicely on 1.7.3 😄
